app/api/v1/endpoints/trading.py

```
@@ -638,11 +834,35 @@ async def get_market_overview(
+            symbol_candidates = await _get_default_market_overview_symbols(user_id=user_id)
+
+        if not symbol_candidates:
+            symbol_candidates = await _get_fallback_market_overview_symbols(
+                limit=MARKET_OVERVIEW_SYMBOL_LIMIT
+            )
+
+        if not symbol_candidates:
+            raise HTTPException(
+                status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+                detail="No market symbols available for overview",
+            )
```

Ruff (0.13.2)

Abstract `raise` to an inner function

(TRY301)